components/datepicker/demo/api.md

```diff
@@ -29,18 +29,18 @@
 
 ## Methods
 
-| Method            | Type                         | Description                                      |
-|-------------------|------------------------------|--------------------------------------------------|
-| [focus](#focus)           | `(focusInput: string): void` | Focuses the datepicker trigger input.<br /><br />**focusInput**: Pass in `endDate` to focus on the return input. No parameter is needed to focus on the depart input. |
-| [pushSlotContent](#pushSlotContent) | `(): void`                   | Emits an event to notify the calendar cells to fetch their slot content. |
+| Method             | Type                         | Description                                      |
+|--------------------|------------------------------|--------------------------------------------------|
+| [focus](#focus)            | `(focusInput: string): void` | Focuses the datepicker trigger input.<br /><br />**focusInput**: Pass in `endDate` to focus on the return input. No parameter is needed to focus on the depart input. |
+| [handleSlotToSlot](#handleSlotToSlot) | `(event: any): void`         |                                                  |
+| [pushSlotContent](#pushSlotContent)  | `(): void`                   | Emits an event to notify the calendar cells to fetch their slot content. |
```

Comment:
Should these be private functions? There is no reason a consumer would call these right?